src/containers/AnalyticsPage/AnaltyicsPage.js

```diff
@@ -1,453 +1,39 @@
-                  </LayoutColumn>
-                  <LayoutColumn width={1 / 3}>
-                    <h5>
-                      {formatMessage({
-                        id: 'container.analyticsPage.tabs.threats.country'
-                      })}
-                    </h5>
-                    {threatsTopCountry}
-                  </LayoutColumn>
-                  <LayoutColumn width={1 / 3}>
-                    <h5>
-                      {formatMessage({
-                        id: 'container.analyticsPage.tabs.threats.type'
-                      })}
-                    </h5>
-                    {threatsTopType}
-                  </LayoutColumn>
-                </LayoutRow>
-                <LayoutRow>
-                  <TimeSeriesChart
-                    xAxisValues={this.getThreatsColumns(analytics)}
-                    yAxisLabels={formatMessage({
-                      id: 'container.analyticsPage.tabs.threats'
-                    })}
-                  />
-                </LayoutRow>
-              </LayoutContainer>
-            </TabsPanel>
-          </Tabs>
-
-          <LayoutRow>
-            <LayoutColumn width={74 / 150}>
-              <AnalyticCard
-                title={formatMessage({
-                  id: 'container.analyticCard.ssl.title'
-                })}
-                description={formatMessage({
-                  id: 'container.analyticCard.duration'
-                })}
-                data={analytics.totals ? analytics.totals.requests.ssl : 0}
-                dataType={formatMessage({
-                  id: 'container.analyticCard.ssl.datatype'
-                })}
-                helpTextId="container.analyticCard.ssl.drawer.help"
-              />
-            </LayoutColumn>
-            <LayoutColumn width={2 / 150}>&nbsp;</LayoutColumn>
-            <LayoutColumn width={74 / 150}>
-              <AnalyticCard
-                title={formatMessage({
-                  id: 'container.analyticCard.bandwidth.title'
-                })}
-                description={formatMessage({
-                  id: 'container.analyticCard.duration'
-                })}
-                data={analytics.totals ? analytics.totals.bandwidth : 0}
-                dataType={formatMessage({
-                  id: 'container.analyticCard.bandwidth.datatype'
-                })}
-                helpTextId="container.analyticCard.bandwidth.drawer.help"
-              />
-            </LayoutColumn>
-          </LayoutRow>
-        </div>
-      </WaitForSettings>
+      <div>
+        <Card>
+          <CardSection>
+            <CardContent title={<FormattedMessage id="container.analyticCard.graphql.header" />}>
+              <p>
+                <FormattedMessage id="container.analyticCard.graphql.overview" />
+              </p>
+            </CardContent>
+            <CardControl>
+              <Button type="primary" onClick={function(){window.open(analyticsDeepLink)}}>
```

Comment:
Lets add 'noopener,noreferrer' to protect the parents opener object.

Reply:
sorted in 13e6fc4